maven-parent/pom.xml

```diff
@@ -403,6 +403,25 @@
             </configuration>
           </execution>
 
+          <execution>
+             <id>Copy and rename frontend configuration</id>
+              <phase>process-resources</phase>
+              <goals>
+                  <goal>run</goal>
+              </goals>
+              <configuration>
+                  <target>
+                      <mkdir dir="${project.build.directory}/${project.artifactId}-${project.version}/distro/configs/openmrs/frontend_config/${project.artifactId}"/>
+                      <copy todir="${project.build.directory}/${project.artifactId}-${project.version}/distro/configs/openmrs/frontend_config/${project.artifactId}">
+                          <fileset dir="${project.basedir}/configs/openmrs/frontend_config" includes="*.json" erroronmissingdir="false"/>
+                      </copy>
+                      <delete>
+                          <fileset dir="${project.build.directory}/${project.artifactId}-${project.version}/distro/configs/openmrs/frontend_config" includes="*.json"/>
```

Comment:
How will the `SPA_CONFIG_URLS` be aware of these?

Should we not add an `.env` file which overrides the `SPA_CONFIG_URLS`?

Reply:
Yes, something like what you've done [here](https://github.com/ozone-his/ozone-docker-compose/pull/176/files#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055dR106). Or alternative is to add Maven task to construct and set `SPA_CONFIG_URLS` based on the contents of `frontend_config/` directory.

Reply:
Yeah.
So that will be a separate PR?